index.Rmd

```diff
@@ -284,13 +288,13 @@ In order to achieve these objectives, you should try to follow a few good practi
     - Indentation (like in Python: use spaces or tab to indent code in functions or loops etc.).
     - Consistent use of the assignment operator: either `<-` or `=` in all your code. The former is used by core R and allows assigning in function calls, the latter is shorter and consistent with most other programming languages.
     - Consistent placement of curly braces.
-* [Make your own packages](#__optional__Writing_packages_).
-* Keep a similar [directory structure](../RProjectManagement/index.html) across your projects.
-* Use [version control](../RProjectManagement/index.html) to develop/maintain your projects and packages.
+* [Make your own packages](#__optional__Writing_packages_). 
+* Keep a similar [directory structure](../RProjectManagement/index.html) across your projects. 
```

Comment:
Oof, we don't have the part about project structure anywhere any more? Then we should probably add it in this tutorial. The project structure is about how you should have a `main.R` file and a `LICENSE` file at the top-level directory, `R` (and/or `Python`) directory with scripts with one file per function, and optional `data` and `output` directories.

Reply:
Yes, the missing content can be found here: https://github.com/GeoScripting-WUR/RProjectManagement/commit/bfb73ccd97695ad51ec680b5db8013941fe7cb6a#diff-e2991a4c746e830eff1ab3566be3720a8ea5111776adc52ff9d85eeaa234cf1a

The part about relative file paths is also very important. No wonder last year students struggled with absolute and relative file paths: we didn't explain them anywhere...

Where would you suggest putting this content? It might be late to put this in this tutorial, as relative file paths are used already in exercise 2. Should we put it into tutorial 1 or tutorial 2?